### ISSUE TITLE
Error when repeating the tutorial for diffusion map in v1.9.1 scanpy

### ISSUE THREAD

**Issue by mmyoung** [scverse/scanpy#2254]
- [ ] I have checked that this issue has not already been reported.
- [ ] I have confirmed this bug exists on the latest version of scanpy.
- [ ] (optional) I have confirmed this bug exists on the master branch of scanpy.

---

**Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.

### This is the code I'm using which was copied from the tutorial:

```python
import scanpy as sc
adata = sc.datasets.paul15()
sc.pp.log1p(adata)
sc.pp.neighbors(adata, n_neighbors=20, use_rep='X', method='gauss')
sc.tl.diffmap(adata)
sc.tl.dpt(adata, n_branchings=1, n_dcs=10)
sc.pl.diffmap(adata, color=['dpt_pseudotime', 'dpt_groups', 'paul15_clusters'])
```
The results I got are:
![image](https://user-images.githubusercontent.com/33322882/168080076-f6767970-0b3b-4623-ab25-e9e38201e6ef.png)

Which is totally different than in the tutorial:
![image](https://user-images.githubusercontent.com/33322882/168081015-cddf77eb-7b83-4802-8890-3b0d87a5755d.png)

Can anyone run into this problem ever? Thanks for your help.

#### Versions

<details>
-----

anndata     0.8.0

scanpy      1.9.1

-----

PIL                         8.4.0

backcall                    0.2.0

beta_ufunc                  NA

binom_ufunc                 NA

bottleneck                  1.3.2

cffi                        1.15.0

colorama                    0.4.4

cycler                      0.10.0

cython_runtime              NA

dateutil                    2.8.2

debugpy                     1.5.1

decorator                   4.4.2

defusedxml                  0.6.0

google                      NA

h5py                        3.6.0

ipykernel                   6.4.1

ipython_genutils            0.2.0

jedi                        0.18.1

joblib                      0.17.0

kiwisolver                  1.3.1

llvmlite                    0.38.0

...

Python 3.9.7 (default, Sep 16 2021, 13:09:58) [GCC 7.5.0]

Linux-3.16.0-11-amd64-x86_64-with-glibc2.19

-----

Session information updated at 2022-05-12 14:59

</details>

**Comment by Koncopd** [comment]
Yes, the same bug as in https://github.com/scverse/scanpy/issues/2227
same quick fix
`adata.obsm["X_diffmap_"] = adata.obsm["X_diffmap"][:, 1:] `
`sc.pl.embedding(adata, "diffmap_")`

**Comment by jolespin** [comment]
@Koncopd do you have any insight on this post? https://discourse.scverse.org/t/how-to-interpret-diffusion-maps/2224

I'm having some trouble understanding how to interpret certain parts of the diffusion maps.
